pallets/elections/src/impls.rs

```diff
@@ -0,0 +1,398 @@
+use crate::{Config, ErasReserved, MembersPerSession, Pallet, SessionValidatorBlockCount};
+use frame_election_provider_support::sp_arithmetic::Perquintill;
+use frame_support::{pallet_prelude::Get, traits::Currency};
+use sp_staking::{EraIndex, SessionIndex};
+use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
+
+const MAX_REWARD: u32 = u32::MAX;
```

Comment:
I'm getting scared that because of possible issues with rounding (and the fact that the staking pallet uses `+` instead of `saturating_add` when handling rewards), we will at some point hit an overflow here, which, no suprise, would be quite bad. For this reason I would go for one of the safe options: 10^9, 2*10^9, or 4*10^9.

Reply:
changed to 10^9